### PR TITLE
4008 hotfix for Sql queries

### PR DESCRIPTION
**Pull request by jmarrec**
Pull request overview
---------------------

 - Followup to #4008 where I mistakenly introduced double WHERE in two statements

### Pull Request Author

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.

 - [ ] Model API Changes / Additions
 - [ ] Any new or modified fields have been implemented in the EnergyPlus ForwardTranslator (and ReverseTranslator as appropriate)
 - [ ] Model API methods are tested (in `src/model/test`)
 - [ ] EnergyPlus ForwardTranslator Tests (in `src/energyplus/Test`)
 - [ ] If a new object or method, added a test in NREL/OpenStudio-resources: Add Link
 - [ ] If needed, added VersionTranslation rules for the objects (`src/osversion/VersionTranslator.cpp`)
 - [ ] Checked behavior in OpenStudioApplication, adjusted policies as needed (`src/openstudio_lib/library/OpenStudioPolicy.xml`)
 - [ ] Verified that C# bindings built fine on Windows, partial classes used as needed, etc.
 - [ ] All new and existing tests passes
 - [ ] If methods have been deprecated, update rest of code to use the new methods

**Labels:**

 - [ ] If change to an IDD file, add the label `IDDChange`
 - [ ] If breaking existing API, add the label `APIChange`
 - [ ] If deemed ready, add label `Pull Request - Ready for CI` so that CI builds your PR

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] Code Style, strip trailing whitespace, etc.
 - [ ] All related changes have been implemented: model changes, model tests, FT changes, FT tests, VersionTranslation, OS App
 - [ ] Labeling is ok
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified

### CODE REVIEW

File: src/utilities/sql/SqlFile_Impl.cpp [hunk 1]
```diff
@@ -849,7 +849,7 @@ namespace openstudio{
       const std::string rowName = t_monthOfYear.valueDescription();
 
 
-      const std::string& s = R"(SELECT Value FROM TabularDataWithStrings WHERE
+      const std::string& s = R"(SELECT Value FROM TabularDataWithStrings
```
Comment (jmarrec) [inline src/utilities/sql/SqlFile_Impl.cpp:852]: First double WHERE

File: src/utilities/sql/SqlFile_Impl.cpp [hunk 2]
```diff
@@ -870,7 +870,7 @@ namespace openstudio{
         " {AT MAX/MIN}";
       const std::string rowName = t_monthOfYear.valueDescription();
 
-      const std::string& s = R"(SELECT Value FROM TabularDataWithStrings WHERE
+      const std::string& s = R"(SELECT Value FROM TabularDataWithStrings
```
Comment (jmarrec) [inline src/utilities/sql/SqlFile_Impl.cpp:873]: Second double where.

File: src/utilities/sql/SqlFile_Impl.cpp [hunk 3]
```diff
@@ -983,24 +983,24 @@ namespace openstudio{
...
+                                  AND Units='GJ')";
       return execAndReturnFirstDouble(s);
     }
 
 
     OptionalDouble SqlFile_Impl::annualTotalCost(const FuelType& fuel) const
     {
       if (fuel == FuelType::Electricity){
-        return execAndReturnFirstDouble("SELECT Value from tabulardatawithstrings where (reportname = 'Economics Results Summary Report') and (ReportForString = 'Entire Facility') and (TableName = 'Annual Cost') and (ColumnName ='Electricity') and (((RowName = 'Cost') and (Units = '~~$~~')) or (RowName = 'Cost (~~$~~)'))");
+        return execAndReturnFirstDouble("SELECT Value from TabularDataWithStrings where (reportname = 'Economics Results Summary Report') and (ReportForString = 'Entire Facility') and (TableName = 'Annual Cost') and (ColumnName ='Electricity') and (((RowName = 'Cost') and (Units = '~~$~~')) or (RowName = 'Cost (~~$~~)'))");
```
Comment (jmarrec) [inline src/utilities/sql/SqlFile_Impl.cpp:1000]: Starting here it's just casing for `tabulardatawithstrings` -> `TabularDataWithStrings`

**Review by joseph-robertson — APPROVED** [review]
👍

**Pull request merged**